src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -207,6 +210,7 @@ private FilePath pidFile(FilePath ws) throws IOException, InterruptedException {
             return controlDir(ws).child("pid");
         }
 
+        // TODO run as one big MasterToSlaveCallable<Integer> to avoid extra network roundtrips
```

Comment:
Irrelevant after #60.

Reply:
Assuming #60 is fundamentally reliable. We can always remove the comment if it tests clean.  And we may have to do yet another emergency fix due to something introduced in #49 before that gets integrated.